crates/krun/src/cli_options.rs

```diff
@@ -0,0 +1,75 @@
+use std::path::PathBuf;
+
+use anyhow::anyhow;
+use bpaf::{any, construct, long, positional, OptionParser, Parser};
+
+use crate::net::NetMode;
+
+#[derive(Clone, Debug)]
+pub struct Options {
+    pub env: Vec<(String, Option<String>)>,
+    pub net: NetMode,
+    pub passt_socket: Option<PathBuf>,
+    pub command: String,
+    pub command_args: Vec<String>,
+}
+
+pub fn options() -> OptionParser<Options> {
+    let env = long("env")
+        .short('e')
+        .help(
+            "Set environment variable to be passed to the microVM
+            ENV should be in KEY=VALUE format, or KEY on its own to inherit the
+            current value from the local environment",
+        )
+        .argument::<String>("ENV")
+        .parse(|s| match s.split_once('=') {
+            Some(("", _)) => Err(anyhow!("invalid ENV format")),
+            Some((k, v)) => Ok((k.to_owned(), Some(v.to_owned()))),
+            None => Ok((s, None)),
+        })
+        .many();
+    let net = long("net")
+        .help(
+            "Set network mode
+            NET_MODE can be either TSI (default) or PASST",
+        )
+        .argument::<String>("NET_MODE")
+        .fallback("TSI".to_owned())
```

Comment:
Please default to PASST, it's more compatible with this use case. TSI is intended to be use with `podman run` (`crun-krun` use case).